### DISCUSSION TITLE
Version 1.0.0 timeline

### DISCUSSION THREAD

**Discussion by jjnrmason** [aws/aws-dotnet-messaging#154] [General]
Hi there, 

Interested in using the library but would need to understand when version `1.0.0` is releasing first. Can you provide a timeline?

**Comment by jjnrmason** [comment]
I had a response for anyone wondering. No planned GA just yet

**Comment by normj** [comment]
That is correct we don't have a date yet for GA. We are leaving the window open to collect feedback in case we need to make breaking changes.
